mysql-test/suite/tianmu/r/alter_table.result

```diff
@@ -1,3 +1,7 @@
+DROP DATABASE IF EXISTS alter_table;
+Warnings:
```

Comment:
If the test case is not intended to display warning, you can disable warning

Reply:
Done.